build/NuSpecs/WindowsAppSDK-Nuget-Native.Bootstrap.targets

```diff
@@ -1,6 +1,6 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <Target Name="GenerateBootstrapCpp" BeforeTargets="ClCompile">
+    <Target Name="GenerateBootstrapCpp">
         <ItemGroup>
             <ClCompile Include="$(MSBuildThisFileDirectory)..\..\include\MddBootstrapAutoInitializer.cpp">
                 <PrecompiledHeader>NotUsing</PrecompiledHeader>
```

Comment:
PrecompiledHeader=NotUsing? was this a workaround for racy addition of compile flags? can we remove it?

Reply:
Oh I remember this - it was a compiler error based on dependencies.  We should keep it.

Reply:
I think it is because we don't know what the name of the precompiled header is, if any, to #include the right file in the cpp, particularly for these pre-defined, non-generated files.